librz/analysis/analysis.c

```diff
@@ -279,6 +279,8 @@ RZ_API bool rz_analysis_set_bits(RzAnalysis *analysis, int bits) {
 		if (analysis->bits != bits) {
 			bool is_hack = is_arm_thumb_hack(analysis, bits);
 			analysis->bits = bits;
+			int v = rz_analysis_archinfo(analysis, RZ_ANALYSIS_ARCHINFO_ALIGN);
+			analysis->pcalign = RZ_MAX(0, v);
```

Comment:
this should be at least 1, not zero.

Reply:
0 means no restrictions at all (which in practice is indeed behaves the same as 1, but the meaning is slightly different). It is checked at all places where it's used so 0 is ok.